Source/Core/UICommon/GameFile.cpp

```diff
@@ -343,6 +357,29 @@ bool GameFile::IsElfOrDol() const
   return name_end == ".elf" || name_end == ".dol";
 }
 
+// Function for finding the folder name two (three if a Wii game) directories up for use in the file
+// name column for extracted games Returns a blank string if the folder can't be found
+std::string GameFile::FindFolderName()
+{
+#ifdef HAS_STD_FILESYSTEM
+  std::filesystem::path game_file_path = StringToPath(m_file_path);
+  game_file_path = game_file_path.parent_path().parent_path();
+  if (m_platform == DiscIO::Platform::WiiDisc)
+  {
+    game_file_path = game_file_path.parent_path();
```

Comment:
It is possible to have an extracted Wii disc without the extra folder level for partitions. This code would fail for such an extracted Wii disc.

Reply:
I believe the best fix for this would be to add an additional check that the folder name is "DATA" before we skip the folder.